### DISCUSSION TITLE
milvus查询时间基本上维持10s-20s之间，查询巨慢，请问是哪里不对呢？

### DISCUSSION THREAD

**Discussion by javayydd** [milvus-io/milvus#27797] [Q&A and General discussion]
![image](https://github.com/milvus-io/milvus/assets/87173166/c04067a4-69fd-4256-a81d-7bfe73745dc0)

![image](https://github.com/milvus-io/milvus/assets/87173166/dafb98a8-09e9-42e4-94f2-7bd49e142a60)

上面是目前测试的向量的存储情况以及新建的索引配置情况
![image](https://github.com/milvus-io/milvus/assets/87173166/a9db910d-7d88-4b72-9ac2-d6bf128cfd60)
这个是我的查询方法，每次查询都得10s以上，请问大佬是哪里配置错了吗？还是查询的方法有问题。
python用的milvus SDK ，版本是2.2.14

**Comment by yhmo** [comment]
collection.load()不用每次search时调用。你是不是search完之后还调用了collection.release()？
